testsuite/integration/clustering/pom.xml

```diff
@@ -535,14 +553,28 @@
                         </goals>
                         <phase>${ts.bootable-jar-packaging.phase}</phase>
                         <configuration>
-                            <output-file-name>test-wildfly-jpa-distributed.jar</output-file-name>
-                            <cli-sessions>
+                            <overwrite-provisioned-server>true</overwrite-provisioned-server>
+                            <record-provisioning-state>false</record-provisioning-state>
+                            <log-provisioning-time>${galleon.log.time}</log-provisioning-time>
+                            <offline-provisioning>${galleon.offline}</offline-provisioning>
+                            <galleon-options>
+                                <jboss-maven-dist/>
+                                <jboss-fork-embedded>${galleon.fork.embedded}</jboss-fork-embedded>
+                                <optional-packages>passive+</optional-packages>
+                            </galleon-options>
+                            <bootableJar>true</bootableJar>
+                            <skipDeployment>true</skipDeployment>
+                            <bootableJarName>test-wildfly-jpa-distributed.jar</bootableJarName>
+                            <packaging-scripts>
                                 <cli-session>
-                                    <script-files>
+                                    <javaOpts>
+                                        <opt>-Dmaven.repo.local=${maven.repo.local}</opt>
+                                    </javaOpts>
```

Comment:
Again, this is more about the plugin. What makes this required? If it is, is this something we should look at adding to the plugin by default?

Reply:
That is not strictly needed there (I didn't run into issue without it). But that is something to generalize. We are starting an embedded server from a slim installation that resolves JBoss Modules modules maven artifact from the maven local cache.